package/src/languageServiceWrapper/kustoLanguageService.ts

```diff
@@ -403,7 +403,10 @@ class KustoLanguageService implements LanguageService {
         let currentCommand = script.GetBlockAtPosition(cursorOffset);
 
         // get completion items
-        const completionItems = currentCommand.Service.GetCompletionItems(cursorOffset);
+        const { includeExtendedSyntax } = this._languageSettings.completionOptions;
+        const completionOptions =
```

Comment:
Not sure how critical, but worth initializing this statically (or at least until the setting changes).
Since run completion runs a lot it might a little costly.